.github/RELEASE_CHECKLIST_TEMPLATE.md

```diff
@@ -14,6 +14,7 @@ Add the issues included in the release and update this document as release tasks
   * nginx/config-files/hydroshare-ssl-nginx.conf.template
   * scripts/templates/docker-compose.template
 - [ ] Changes in hydroshare/local_settings.py need to be coordinated for manual update
+- [ ] Smoke test active worker before swap, note whether maintenance will be needed
 - [ ] Deployed to Beta
```

Comment:
Should we also include a check for running any management command after the deploy?